x/auth/middleware/basic_test.go

```diff
@@ -0,0 +1,222 @@
+package middleware_test
+
+import (
+	"strings"
+
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/cosmos/cosmos-sdk/crypto/types/multisig"
+	"github.com/cosmos/cosmos-sdk/testutil/testdata"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	"github.com/cosmos/cosmos-sdk/x/auth/middleware"
+	"github.com/tendermint/tendermint/abci/types"
+)
+
+func (suite *MWTestSuite) TestValidateBasic() {
```

Comment:
I noticed it's sometimes suite, sometimes s, let's maybe just use s everywhere.

```suggestion
func (s *MWTestSuite) TestValidateBasic() {
```